proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -91,14 +91,9 @@ service BeaconChain {
         };
     }
 
-    // Retrieve blocks by root, slot, or epoch.
-    //
-    // The server may return multiple blocks in the case that a slot or epoch is
-    // provided as the filter criteria. The server may return an empty list when
-    // no blocks in their database match the filter criteria. This RPC should
-    // not return NOT_FOUND. Only one filter criteria should be used. This endpoint
-    // allows for retrieval of genesis information via a boolean query filter.
+    // DEPRECATED: Do not use.
```

Comment:
Even though it's deprecated, I think we should still leave the documentation explaining what it does. You can format it as:

```
// DEPRECATED: Do not use.
//
// [Documentation]